README.md

```diff
@@ -604,6 +605,35 @@ This library partially supports [inline schema definition dereferencing]( http:/
 
 Note that it only supports local definition referencing, we do not plan on fetching foreign schemas over HTTP anytime soon. Basically, you can only reference a definition from the very schema object defining it.
 
+## Troubleshooting
+
+### Build error: Cannot find module 'buffertools' from  .../node_modules/react-jsonschema-form/node_modules/deeper
+
+This is [an issue](https://github.com/othiym23/node-deeper/issues/3#issuecomment-206232232) in the way `deeper` handles the optional `buffertools` dependency (by using try/catch around a require('buffertools')).
+
+If you use **webpack** you can do the following to ignore this import:
+
+~~~javascript
```

Comment:
Please use the same md syntax or code blocks, as used everywhere else in the file for consistency.